app/src/main/java/com/hedvig/app/feature/marketpicker/MarketPickerActivity.kt

```diff
@@ -82,6 +82,7 @@ class MarketPickerActivity : BaseActivity(R.layout.activity_market_picker) {
 
         save.setHapticClickListener {
             model.save()
+            finish()
```

Comment:
Is this safe?

Should we use the method we usually use with `Intent.FLAG_ACTIVITY_NEW_TASK` + `Intent.FLAG_ACTIVITY_CLEAR_TASK`?